stumpy/gpu_aamp.py

```diff
@@ -10,6 +10,7 @@
 from numba import cuda
 
 from . import core, config
+from .core import _gpu_searchsorted_right
```

Comment:
It almost feels like you don't need this line and then later, you'd do

`core._gpu_searchsorted_right`

Reply:
Yeah...the only reason I did that was to avoid the very little time needed to look up the function  `_gpu_searchsorted_right` in the `core.py`. Since `_gpu_searchsorted_right` can be called a lot of times throughout the process, I thought it might improve performance a little bit 😄 

I will do `core._gpu_searchsorted_right`